### DISCUSSION TITLE
Puan - Coin ilişkisi

### DISCUSSION THREAD

**Discussion by bitcotrader** [IBAX-io/go-ibax#601] [Ideas]
Puanların coin karşılığı ne olacak veya nasıl bir dağıtım yapılacak belirlenmesi gerekir. Twitter paylaşımında insanlar coin gelirlerini merak ediyor, puan dendiği zaman fazla etkileşim alınmıyor.

**Comment by hawkins09** [comment]
puan sadece airdrop çekilişinde avantaj sağlamak için herhangi bi maddiyatı yok airdrop sonucunda kazandığımız ibxc coini önemli

**Comment by utku3755** [comment]
ibcx i fazla kazanmak mumkun mü retweet haricinde
